build.sbt

```diff
@@ -31,13 +30,13 @@ lazy val root = (project in file("."))
       Cats.core,
       Cats.effect,
       Cats.log4cats,
-      Cats.log4catsSlf4j,
+      Cats.log4catsSlf4j  % Test,
       Fs2.core,
       Fs2.kafka,
-      embeddedKafka,
-      scalaTest,
-      catsEffectTesting,
-      logbackClassic
+      catsEffectTesting   % Test,
+      logbackClassic      % Test,
```

Comment:
Same as above, we shouldn't assume the logging backend, and this is only needed for tests